server/report_builder/src/build.rs

```diff
@@ -101,8 +99,8 @@ fn make_report(args: &BuildArgs, mut files: HashMap<String, PathBuf>) -> Result<
         header: None,
         footer: None,
         query: vec![],
-        convert_data: None,
-        custom_wasm_function: None,
+        // Convert data is generated outside of this method call
```

Comment:
Everything will be moved to cli eventually, and i couldn't share ConvertDataType with report_builder, plus wanted to use the utility method to check for bad status